OpenRA.Mods.Common/Traits/BlocksProjectiles.cs

```diff
@@ -13,6 +13,11 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public interface IBlocksProjectiles
+	{
+		WDist GetHeight(Actor self);
```

Comment:
Traits would get the height directly from the traits info before, that's why it's just simply a getter now. Suggestions on a better name?